packages/expo-av/src/Video.types.ts

```diff
@@ -215,6 +215,11 @@ export type VideoProps = {
    * See the [AV documentation](./av) for more information.
    */
   isMuted?: boolean;
+  /**
+   * The desired audio panning value of the audio for this media. This value must be between `-1.0` (full left) and `1.0` (full right).
+   * See the [AV documentation](./av) for more information.
+   */
+  audioPan?: number;
```

Comment:
Does the volume pan work on video too? :D

Reply:
> Does the volume pan work on video too? :D

It does, of course, only with the MediaPlayer implementation